sport.map

```diff
@@ -0,0 +1,286 @@
+#==============================================================================
+#
+# sport.map
+# omschrijving: sportfaciliteiten en aanbieders
+# datasets:
+# (1) zwembaden
+# (2) hardlooproutes
+# (3) aanbieders
+# (4) openbare sportplekken
+# (5) sportparken
+# (6) sporthallen
+# (7) sportvelden
+# (8) gym- en sportzalen
+#==============================================================================
+
+MAP
+  NAME                      "sport"
+  INCLUDE                   "header.inc"
+  DEBUG 1
+
+  WEB
+    METADATA
+      "ows_title"           "sport"
+      "ows_abstract"        "sport"
+      "wms_extent"          "100000 450000 150000 500000"
+    END
+  END
+
+  LEGEND
+       STATUS ON
+       KEYSIZE 17 17
+  END
+
+
+   #-----------------------------------------------------------------------------
+  
+    LAYER
+    NAME            "sport"
+    GROUP           "sport"
```

Comment:
De verschillende lagen moeten ook verschillende namen hebben. Dan kunnen ze afzonderlijk getoond worden. Ik zou deze laag zwembaden noemen 
De group naam sport is wel ok. Dan kan alles  aan worden gezet door de hele group te selecteren

Reply:
Check